### PR TITLE
Issue #55 menu implementation

### PR DESCRIPTION
**Pull request by Alex-Hayeur**
- Adds settings menu link to the home and profile page allowing user to log out.
- Adds favourites button and add listing button to bottom navigation menu.
- Stops bottom menu from shifting icons when one is selected.

### CODE REVIEW

**Comment by davidberard2** [comment]
It seems really good, I've had the opportunity to look at the app and everything seems to work perfectly well. Did you plan to implement the followings in the future to make sure it respond to the illustration Karen did? Is it possible to implement the + as on the drawing and make it dessapear when not logged in ? (the 3 lines should be the wheel) 

screen shot 2017-10-25 at 10 21 29 am

screen shot 2017-10-25 at 10 10 07 am

**Comment by danielanthonyr** [comment]
I agree with David's review, once that is done, then it will look amazing!

**Comment by LHuss** [comment]
I like it. Maybe we can merge the request now so that people working on other features can work on them, and the additional changes can be merged at a later date.

File: app/build.gradle [hunk 1]
```diff
@@ -37,7 +37,7 @@ dependencies {
     compile 'com.android.support:design:26.+'
     compile 'com.android.support:recyclerview-v7:23.1.1'
     compile 'com.android.support:support-v4:26.+'
-    compile 'com.android.support:cardview-v7:21.0.+'
+    compile 'com.android.support:cardview-v7:21.0.3'
```
Comment (danielanthonyr) [inline app/build.gradle:40]: This should be compile 'com.android.support:cardview-v7:21.0.+'

**Review by davidberard2 — APPROVED** [review]

**Review by danielanthonyr — APPROVED** [review]

**Pull request merged**